packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.test.js

```diff
@@ -16,7 +16,7 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
 
-import { NONE, DURATION, TAG } from '../settings/SpanBarSettings';
+import { NONE, DURATION, TAG } from 'app/core/components/SpanBarSettings/SpanBarSettings';
```

Comment:
This should not be imported from here as they are in `packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx`